spec/data_migrations/delete_nil_evidences_spec.rb

```diff
@@ -34,8 +34,7 @@
     end
 
     before do
-      applicant.evidences << FinancialAssistance::Evidence.new
-      applicant.save!
+      applicant.set(evidences: [FinancialAssistance::Evidence.new])
```

Comment:
This is added because we have to skip the callbacks to test it since we added a callback